DeepSkyStacker/DSSVersion.h

```diff
@@ -1,7 +1,7 @@
 #ifndef __DSSVERSION_H_
 #define __DSSVERSION_H_
 
-#define DSSBETA
+//#define DSSBETA
```

Comment:
I am not sure if this is meant to stay uncommented, but when I tried running the project with this macro on I got an error message "This beta version has expired. You can probably get another one or download the final release from the web site.", so I commented it.